tests/resources/modules/d8/woot/src/WootCommandInfoAlterer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\woot;
+
+use Consolidation\AnnotatedCommand\CommandInfoAltererInterface;
+use Consolidation\AnnotatedCommand\Parser\CommandInfo;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+
+class WootCommandInfoAlterer implements CommandInfoAltererInterface, LoggerAwareInterface
+{
+    use LoggerAwareTrait;
```

Comment:
You can drop the LoggerAwareInterface / LoggerAwareTrait here. Drush will only inject its logger into command classes that implement these interfaces -- not any object at all. Since you are instantiating this class via the Drupal service file (drush.services.yml), then Drupal will inject your logger factory, so the code you have below is sufficient.

Reply:
OK, removed the trait and interface.